test/Microcosm-test.js

```diff
@@ -106,4 +106,16 @@ describe('Microcosm', function() {
     })
   })
 
+  it ('throws errors by default if a callback is not provided to start', function() {
+    let app = new Microcosm()
+    let error = 'This error should exist!'
+
+    app.addPlugin(function willPassError (app, options, next) {
+      next(error)
+    })
+
+    assert.throws(function() {
+      app.start()
+    }, error)
```

Comment:
I'm not sure I follow this test. Why do you need to add a plugin just to test that `app.start()` w/out a callback will throw an error?